buildbot/master.cfg

```diff
@@ -576,6 +577,13 @@ def make_android_package(mode="normal"):
                                descriptionDone="Gradle",
                                haltOnFailure=True))
 
+        master_filename = WithProperties("/srv/http/dl/prs/dolphin-%s.apk", "branchname")
+        url = WithProperties("http://dl.dolphin-emu.org/prs/dolphin-%s.apk", "branchname")
+        
+        f.addStep(FileUpload(slavesrc="Source/Android/app/build/outputs/apk/app-arm_64-release.apk",
```

Comment:
The modes are 'normal', 'pr', and 'wip'. Not sure what 'wip' is used for, though.